resources/main.css

```diff
@@ -2,17 +2,41 @@
     --viewport-width: 800px;
     --viewport-height: 600px;
     --foreground: rgb(235, 235, 235);
+    --inactive-color: rgb(128, 128, 128);
```

Comment:
I like the use of custom properties and it seems that we're introducing these progressively. 
Right now it feels slightly inconsistent between colors for example. 
This might be out of scope here, but maybe we can address a consistent use in a separate pr?

Reply:
Let me create a PR for the last color we're missing.